### ISSUE TITLE
Error when sending photo

### ISSUE THREAD

**Issue by petorian** [windkh/node-red-contrib-telegrambot#143]
When I try to send a photo, I get the following error:
"TypeError: Cannot read property 'sendPhoto' of undefined"

The message is constructed like this:
msg.payload.content = '/etc/openhab2/html/lastsnap.jpg';
msg.payload.type = 'photo';
msg.payload.chatId = '-1xxxxxxxx';

/* type can be one of the following
photo
audio
video
sticker
voice
document
*/

return msg;

**Comment by windkh** [comment]
Does the send photo example flow in the examples folder work?

**Comment by windkh** [comment]
I just tried it and ot works using version 8.8.0
![image](https://user-images.githubusercontent.com/10910767/99283907-e8db5c80-2835-11eb-90c7-406942a5a4f6.png)

![image](https://user-images.githubusercontent.com/10910767/99283952-f7297880-2835-11eb-9dfb-ee28426c787e.png)

What version do you use?

**Comment by petorian** [comment]
I'm using version 8.8.0 as well. 
And version 1.2.3 of Node-Red (installed on Openhabian)

**Comment by windkh** [comment]
Hm... can you attach a debugger to the node red instance?

**Comment by petorian** [comment]
I had also a different problem; every time I tried to change anything on the flows, the Deploy button would not respond. I followed this thread: https://discourse.nodered.org/t/deploy-button-not-working/2972/30 and ended up deleting the JSON files and started over with a blank canvas. If I then use the provided example it works perfect. If interested I can share the faulty json file with you windkh, but for me the problem is now solved. Thanks for your quick response.
